src/app/code/local/Omikron/Factfinder/Helper/Upload.php

```diff
@@ -35,7 +35,7 @@ public function upload($sourcePath, $destinationPath)
                     [
                         'host' => $this->getConfig(self::PATH_FF_UPLOAD_HOST),
                         'user' => $this->getConfig(self::PATH_FF_UPLOAD_USER),
-                        'password' => $this->getConfig(self::PATH_FF_UPLOAD_PASSWORD),
+                        'password' => Mage::helper('core')->decrypt($this->getConfig(self::PATH_FF_UPLOAD_PASSWORD)),
```

Comment:
If you add `<ff_upload_password backend_model="adminhtml/system_config_backend_encrypted" />` to the `config.xml` there is no need to decrypt manually.

Reply:
@fmarangi It's fixed